### PR TITLE
Add an initial CODEOWNERS

### PR DESCRIPTION
**Pull request by dnephin**
If anyone would like to be added to a path just let me know and I can update this PR.

### CODE REVIEW

**Comment by vieux** [comment]
LGTM it's a good start, ping @tonistiigi @thaJeztah

**Comment by tonistiigi** [comment]
LGTM

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Github code owners
+# See https://help.github.com/articles/about-codeowners/
+
+builder/**          @dnephin @tonistiigi
+hack/**             @dnephin
```
Comment (thaJeztah) [inline .github/CODEOWNERS]: Perhaps @tianon ?

Comment (thaJeztah) [inline .github/CODEOWNERS]: (also for https://github.com/moby/moby/tree/master/contrib/mkimage)

**Review by thaJeztah — COMMENTED** [review]
Some other suggestions;

- @rhvgoyal for https://github.com/moby/moby/tree/master/daemon/graphdriver/devmapper
- @dmcgowan for https://github.com/moby/moby/tree/master/daemon/graphdriver/overlay2, and https://github.com/moby/moby/tree/master/daemon/graphdriver/overlay
- @johnstep and @jhowardmsft (?) for https://github.com/moby/moby/tree/master/daemon/graphdriver/windows
- same for https://github.com/moby/moby/tree/master/daemon/graphdriver/lcow
- @cpuguy83 for https://github.com/moby/moby/tree/master/plugin

**Comment by thaJeztah** [comment]
Oh, and @samuelkarp for https://github.com/moby/moby/tree/master/daemon/logger/awslogs (I can dig into some more of the logging drivers)

**Comment by AkihiroSuda** [comment]
me for https://github.com/moby/moby/tree/master/hack/integration-cli-on-swarm ?

**Comment by dnephin** [comment]
Perfect, I've added all those suggestions. We don't have to add everyone at once. Let's merge the current list if it looks correct? We can add more in follow up PRs

**Review by thaJeztah — APPROVED** [review]
sounds like a plan 👍 LGTM

thanks for doing this!

**Comment by thaJeztah** [comment]
no need to wait for CI; merging

**Pull request merged**